index.md

```diff
@@ -291,6 +291,105 @@ This is an overview of all Android versions and their corresponding identifiers
 </table>
 </div>
 
+## Target version policy
+
+#### Policy rules
+<div class="table-responsive">
+<table class="full-width">
+  <tr>
+    <th class="nowrap">Name</th>
+    <th>Definition</th>
+  </tr>
+  <tr>
+    <td class="nowrap"><code>new app</code></td>
+    <td>Must target an android api level <strong>within one year</strong> from the latest major Android OS version.</td>
+  </tr>
+  <tr>
+    <td class="nowrap"><code>update app</code></td>
+    <td>Must target an android api level <strong>within one year</strong> from the latest major Android OS version.</td>
+  </tr>
+  <tr>
+    <td class="nowrap"><code>existing app</code></td>
+    <td>Must target an android api level <strong>within two years</strong> from the latest major Android OS version. Otherwise you have to expect <a href="https://android-developers.googleblog.com/2022/04/expanding-plays-target-level-api-requirements-to-strengthen-user-security.html">restrictions</a>.</td>
+  </tr>
+  <tr>
+    <td class="nowrap"><code>wear os app</code></td>
+    <td>Must target an android api level <strong>within two years</strong> from the latest major Android OS version. Otherwise you have to expect <a href="https://android-developers.googleblog.com/2022/04/expanding-plays-target-level-api-requirements-to-strengthen-user-security.html">restrictions</a>.</td>
+  </tr>
+  <tr>
+    <td class="nowrap"><code>automobil app</code></td>
+    <td>Must target an android api level <strong>within two years</strong> from the latest major Android OS version. Otherwise you have to expect <a href="https://android-developers.googleblog.com/2022/04/expanding-plays-target-level-api-requirements-to-strengthen-user-security.html">restrictions</a>.</td>
+  </tr>
+</table>
+</div>
+
+#### Required targetSdkVersion for your app in the Google Play Store
+
+This is a list that shows which <code>targetSdkVersion</code> your app needs in the Google Play Store, without any <a href="https://android-developers.googleblog.com/2022/04/expanding-plays-target-level-api-requirements-to-strengthen-user-security.html">restrictions</a>.
+
+<div class="table-responsive">
+<table class="full-width">
+  <tr>
+    <th>SDK / API level</th>
```

Comment:
I think since the "key" of the table is still API level, this should be in the big table. Would a nested table work?

Reply:
@TWiStErRob I created on purpose a new table because the big one would become to big and messi.